languages/csharp/exercises/concept/enums-advanced/.docs/introduction.md

```diff
@@ -1 +1,26 @@
-The C# `enum` type represents a fixed set of named constants (an enumeration). Normally, one can only refer to exactly one of those named constants. However, sometimes it is useful to refer to more than one constant. To do so, one can mark the `enum` as one which constants are _flags_. By carefully assigning the values of each constant, one can use bitwise operators to add or remove references to one or more of the (flag) constants.
+The C# `enum` type represents a fixed set of named constants (an enumeration). Normally, one can only refer to exactly one of those named constants. However, sometimes it is useful to refer to more than one constant. To do so, one can mark the `enum` as one which constants are _flags_. By carefully assigning the values of each constant, one can use bitwise operators to add or remove references to one or more of the (flag) constants. A flags enum can be defined as follows:
+
+```csharp
+[Flags]
+public enum PhoneFeatures
+{
+    Call = 1,
+    Text = 2
+}
+```
+
+To work with bits, C# supports the following operators:
```

Comment:
While the paragraph above does mention that bitwise operators can be used, I feel that it isn't really explained that flags means the constants are arranged in memory as bits. This might be the first mention of bits a learner sees, can the link between flags and bits be explained a bit more before introducing the operators?

Reply:
Agreed. I've updated the text a bit, and also added a part about changing the backing type (which I forgot to add previously).